README.md

```diff
@@ -168,18 +168,62 @@ return fmt.Errorf("invalid port: %v", err)
 
   ```
   Note: Only the description comments of patches that actually **apply** are displayed.
+- `cd` to your Go project's directory.
+
+  ```shell
+  $ cd ~/go/src/example.com/myproject
+  ```
+
+  Run `gopatch` on the project, supplying the previously written patch with the
+  `-p` flag along with '--print-only' flag.
+
+  ```shell
+  $ gopatch --print-only -p ~/s1028.patch ./...
+  ```
+
+  This will turn on print-only mode and will print the changed code stdout instead of modifying
+  all the Go code in your project. To provide more context on what the patch does, if
```

Comment:
We should not show --print-only for the intro tutorial.
The output will be meaningless to people because it'll just print their Go files.
`--diff` during the intro makes sense as a "use --diff to see the changes gopatch is about to make".